<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocksOptionsClass</class>
 <widget class="QWidget" name="SocksOptionsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>492</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbSocksStreams">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="grbDirectConnection">
        <property name="title">
         <string>Incoming Direct Connections</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="pblPort">
             <property name="text">
              <string>Listened port:</string>
             </property>
             <property name="buddy">
              <cstring>spbPort</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spbPort">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chbDisableDirectConnect">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Disable direct connections</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="grbForward">
           <property name="title">
            <string>Port Forwarding</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblForwardHost">
              <property name="text">
               <string>Host:</string>
              </property>
              <property name="buddy">
               <cstring>lneForwardHost</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lneForwardHost"/>
            </item>
            <item>
             <widget class="QLabel" name="pblForwardPort">
              <property name="text">
               <string>Port:</string>
              </property>
              <property name="buddy">
               <cstring>spbForwardPort</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spbForwardPort">
              <property name="maximum">
               <number>65536</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grbStreamProxy">
        <property name="title">
         <string>Stream Proxy</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="chbUseNativeServerProxy">
           <property name="text">
            <string>Use proxy on native server</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLineEdit" name="lneStreamProxy"/>
             </item>
             <item>
              <widget class="QListWidget" name="ltwStreamProxy">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="pbtAddStreamProxy">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbtStreamProxyUp">
               <property name="text">
                <string>Up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbtStreamProxyDown">
               <property name="text">
                <string>Down</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbtDeleteStreamProxy">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grbProxy">
        <property name="title">
         <string>Connection Proxy</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="4">
          <widget class="QCheckBox" name="chbUseAccountProxy">
           <property name="text">
            <string>Use account connection proxy settings</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblProxyType">
           <property name="text">
            <string>Type:</string>
           </property>
           <property name="buddy">
            <cstring>cmbProxyType</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QComboBox" name="cmbProxyType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblProxyHost">
           <property name="text">
            <string>Host:</string>
           </property>
           <property name="buddy">
            <cstring>lneProxyHost</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lneProxyHost"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="lblProxyPort">
           <property name="text">
            <string>Port:</string>
           </property>
           <property name="buddy">
            <cstring>spbProxyPort</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="spbProxyPort">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblProxyUser">
           <property name="text">
            <string>Username:</string>
           </property>
           <property name="buddy">
            <cstring>lneProxyUser</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QLineEdit" name="lneProxyUser"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="pblProxyPassword">
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="buddy">
            <cstring>lneProxyPassword</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="3">
          <widget class="QLineEdit" name="lneProxyPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spbPort</tabstop>
  <tabstop>chbDisableDirectConnect</tabstop>
  <tabstop>lneForwardHost</tabstop>
  <tabstop>spbForwardPort</tabstop>
  <tabstop>chbUseNativeServerProxy</tabstop>
  <tabstop>lneStreamProxy</tabstop>
  <tabstop>ltwStreamProxy</tabstop>
  <tabstop>pbtAddStreamProxy</tabstop>
  <tabstop>pbtStreamProxyUp</tabstop>
  <tabstop>pbtStreamProxyDown</tabstop>
  <tabstop>pbtDeleteStreamProxy</tabstop>
  <tabstop>chbUseAccountProxy</tabstop>
  <tabstop>cmbProxyType</tabstop>
  <tabstop>lneProxyHost</tabstop>
  <tabstop>spbProxyPort</tabstop>
  <tabstop>lneProxyUser</tabstop>
  <tabstop>lneProxyPassword</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chbDisableDirectConnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>spbPort</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbDisableDirectConnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>grbForward</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbUseAccountProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbProxyType</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbUseAccountProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>spbProxyPort</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbUseAccountProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>lneProxyHost</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbUseAccountProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>lneProxyUser</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbUseAccountProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>lneProxyPassword</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
